activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/web/controller/IdentityManagementController.java

```diff
@@ -66,7 +66,7 @@ public List<Group> getGroups(@RequestParam(value = "search", required = false) S
     }
 
     @RequestMapping(value = "/roles", method = RequestMethod.GET)
-    public UserRoles getUserRoles(@AuthenticationPrincipal Principal principal) {
+    public UserRoles getUserRoles(@AuthenticationPrincipal Jwt principal) {
```

Comment:
@jesty Changed to a Jwt object to prevent the principal from being `null`.
As per the Javadocs, if the currently authenticated user does not match in format with the class marked as `@AuthenticationPrincipal`, the principal resolves to `null`.
See: https://docs.spring.io/spring-security/site/docs/4.0.3.RELEASE/apidocs/org/springframework/security/web/bind/support/AuthenticationPrincipalArgumentResolver.html